docs/customization/menu.rst

```diff
@@ -274,6 +267,11 @@ listener to the ``sylius.menu.admin.order.show`` event in the ``config/services.
             tags:
                 - { name: kernel.event_listener, event: sylius.menu.admin.order.show, method: addAdminOrderShowMenuItems }
 
+After these two steps your admin panel order menu should look like that, the new item appears at right corner:
```

Comment:
After these two steps, your admin panel order menu should look like that (the new item appears at the right corner):